storage/list.go

```diff
@@ -0,0 +1,79 @@
+package storage
+
+import (
+	"encoding/json"
+	"fmt"
+	"log/slog"
+	"net/http"
+	"os"
+	"text/tabwriter"
+	"time"
+
+	"github.com/fewsats/fewsatscli/client"
+	"github.com/urfave/cli/v2"
+)
+
+type File struct {
+	ID              uint64    `json:"id"`
+	ExternalID      string    `json:"external_id"`
+	UserID          uint64    `json:"user_id"`
+	Name            string    `json:"name"`
+	Description     string    `json:"description"`
+	Size            uint64    `json:"size"`
+	Extension       string    `json:"extension"`
+	MimeType        string    `json:"mime_type"`
+	StorageUrl      string    `json:"storage_url"`
+	PriceInUsdCents uint64    `json:"price_in_usd_cents"`
+	CreatedAt       time.Time `json:"created_at"`
+	UpdatedAt       time.Time `json:"updated_at"`
+}
+
+var listCommand = &cli.Command{
+	Name:   "list",
+	Usage:  "List all files.",
+	Action: listFiles,
+}
+
+func printFiles(files []File) {
```

Comment:
I will check how other tools do I because I like quite a lot the ones that let you specify the output type `--json` for example